templates/flavors/rke2/full/kustomization.yaml

```diff
@@ -71,3 +72,19 @@ patches:
                   device: /dev/sdc
               mounts:
                 - ["/dev/sdc", "/var/lib/etcd_data"]
+  - target:
+      group: infrastructure.cluster.x-k8s.io
+      version: v1alpha2
+      kind: LinodeMachineTemplate
+    patch: |-
+      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+      kind: LinodePlacementGroup
```

Comment:
question: Is this suppose to be kind: LinodePlacementGroup? Wouldn't this be LinodeMachineTemplate?